README.md

```diff
@@ -76,12 +75,12 @@ my_s3_folder = my_cluster_folder.to("s3", path="my_bucket/my_folder")
 my_local_folder = my_s3_folder.to("here")
 ```
 
-You can send a function to the cluster, including the environment in which the function will live, which is actually
+You can send a [function](https://www.run.house/docs/api/python/function#function) to the cluster, including the environment in which the function will live, which is actually
 set up in its own Ray process. You can send it to an existing env, or create a new one on the fly. Like the folder
-above, the function object which is returned from `.to` is a proxy to the remote function. When you call it a
+above, the function object which is returned from `.to` is a proxy to the remote function. When you call it, a
 lightweight request is sent to the cluster's Runhouse HTTP server to execute the function with the given inputs and
 returns the results. Note that the function is not serialized, but rather imported on the cluster after the local
-working directory (`"./"`, by default the git root) is sent up.
+working directory (`"./"`, by default the git root) is sent up. This results in a significant speed reduction vs. the serialized approach.
```

Comment:
This is not correct, it's about package version mismatches. Probably better to just drop this.

Reply:
Removed.
Should we mention what are the benefits of the non-serialized approach? Otherwise it's an implementation detail which might be unclear to the reader at this point?